ne/README.md

```diff
@@ -44,37 +44,36 @@ JavaScript is an interpreted language. While running Javascript an interpreter i
 
 ### Code, and what to do with it
 
 ### कोड, र यसको साथ के गर्ने
 
 _Code_ is the written instructions that make up a program. Many chapters in this book contain a lot of code, and it is important to read and write code as part of learning how to program. You should not just quickly scan the examples - read them carefully and try to understand them. This may be difficult at first, but with practice, you will improve. The same goes for the exercises - make sure you actually try to write a solution before assuming you understand them. It is also helpful to try running your solutions to the exercises in a JavaScript interpreter, as this will allow you to see if your code is working correctly and may encourage you to experiment and go beyond the exercises.
 
-_कोड_ लिखित निर्देशन हो जसले कार्यक्रम बनाउँदछ। यस पुस्तकका धेरै अध्यायहरूमा धेरै कोडहरू छन्, र कसरी प्रोग्राम गर्ने भनेर सिक्ने भागको रूपमा कोड पढ्न र लेख्न महत्त्वपूर्ण छ। तपाईंले उदाहरणहरू तुरुन्तै स्क्यान गर्नुहुँदैन - तिनीहरूलाई ध्यानपूर्वक पढ्नुहोस् र तिनीहरूलाई बुझ्ने प्रयास गर्नुहोस्। यो सुरुमा गाह्रो हुन सक्छ, तर अभ्यासको साथ, तपाईं सुधार हुनेछ। व्यायामको लागि पनि त्यस्तै हुन्छ - सुनिश्चित गर्नुहोस् कि तपाईं वास्तवमा तिनीहरूलाई बुझ्नु अघि समाधान लेख्ने प्रयास गर्नुहुन्छ। यो जाभास्क्रिप्ट दुभाषियामा अभ्यासहरूको लागि तपाईंको समाधानहरू चलाउन प्रयास गर्न पनि उपयोगी छ, किनकि यसले तपाईंलाई तपाईंको कोडले सही तरिकाले काम गरिरहेको छ कि छैन भनेर हेर्न अनुमति दिन्छ र तपाईंलाई प्रयोग गर्न र अभ्यासहरू भन्दा बाहिर जान प्रोत्साहित गर्न सक्छ।
-
+_कोड_ लिखित निर्देशन हो जसले प्रोग्राम बनाउँदछ। यस पुस्तकका धेरै अध्यायहरूमा धेरै कोडहरू छन्, र कसरी प्रोग्राम गर्ने भनेर सिक्ने भागको रूपमा कोड पढ्न र लेख्न महत्त्वपूर्ण छ। तपाईंले उदाहरणहरू तुरुन्तै स्क्यान गर्नुहुँदैन - तिनीहरूलाई ध्यानपूर्वक पढ्नुहोस् र तिनीहरूलाई बुझ्ने प्रयास गर्नुहोस्। यो सुरुमा गाह्रो हुन सक्छ, तर अभ्यासको साथ, तपाईं सुधार हुनेछ। व्यायामको लागि पनि त्यस्तै हुन्छ - सुनिश्चित गर्नुहोस् कि तपाईं वास्तवमा तिनीहरूलाई बुझ्नु अघि समाधान लेख्ने प्रयास गर्नुहुन्छ। यो जाभास्क्रिप्ट दुभाषियामा अभ्यासहरूको लागि तपाईंको समाधानहरू चलाउन प्रयास गर्न पनि उपयोगी छ, किनकि यसले तपाईंलाई तपाईंको कोडले सही तरिकाले काम गरिरहेको छ कि छैन भनेर हेर्न अनुमति दिन्छ र तपाईंलाई प्रयोग गर्न र अभ्यासहरू भन्दा बाहिर जान प्रोत्साहित गर्न सक्छ।
 
 ### Typographic conventions
 
 ### टाइपोग्राफिक कन्भेन्सनहरू
 
 In this book, text written in a monospaced font represents elements of a program. This can be a self-contained fragment or a reference to part of a nearby program. Programs, like the one shown below, are written in this way:
 
-यस पुस्तकमा एक मोनोस्पेड फन्टमा लेखिएका पाठले कार्यक्रमको तत्वहरू प्रतिनिधित्व गर्दछ। यो एक स्व-निहित अंश वा नजिकैको कार्यक्रमको अंशको सन्दर्भ हुन सक्छ।
-एउटा उदाहरण  तल देखाइएको छ।
+यस पुस्तकमा एक मोनोस्पेड फन्टमा लेखिएका पाठले प्रोग्राम को तत्वहरू प्रतिनिधित्व गर्दछ। यो एक स्व-निहित अंश वा नजिकैको प्रोग्राम को अंशको सन्दर्भ हुन सक्छ।
```

Comment:
Do we need to have a space here (`प्रोग्राम को`)?

Reply:
no we don't

Reply:
other replacements are complete